lib/PrintfPass.cpp

```diff
@@ -97,11 +97,13 @@ void clspv::PrintfPass::DefinePrintfInstance(Module &M, CallInst *CI,
   auto *ConstantNegOne = ConstantInt::get(Int32Ty, -1);
   auto *ConstantPrintfID = ConstantInt::get(Int32Ty, PrintfID);
   auto *ConstantAllocSize = ConstantInt::get(Int32Ty, AllocSizeInts);
+  auto *ConstantArgsSize = ConstantInt::get(Int32Ty, ArgsSize / 4);
   auto *ConstantBuffSize =
-      ConstantInt::get(Int32Ty, clspv::Option::PrintfBufferSize());
+      ConstantInt::get(Int32Ty, clspv::Option::PrintfBufferSize() / 4);
```

Comment:
The buffer size is used to compare it with the printf counter.
This counter counts the number of word (32bit element) written, thus we need to consider the size as a number of word